css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+body {
+  background-color: #0090e0;
+  font-family: Lato, Arial, Helvetica, sans-serif;
+}
+
+.container {
+  max-width: 1000px;
+  margin: auto;
+  background-color: #0090e0;
+  color: white;
+}
+
+.column-third {
+  height: 110px;
+  width: 110px;
+  margin: 10px 90px;
+  padding: -30px;
+  border-radius: 50%;
+  background-color: white;
+}
+
+img {
+  width: 200px;
+  padding: 70px;
+  margin-top: -42px;
+  margin-bottom: -40px;
+  margin-left: -43px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  display: flex;
```

Comment:
`column` classes should not be flexboxes. If you want the `column` to be a flexbox, that typically means you want a `row` inside the `column`